### PR TITLE
added intial schema migrations

### PR DESCRIPTION
**Pull request by jastribl**

### CODE REVIEW

File: html_helper.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+let cheerio = require('cheerio');
+let fs = require('fs');
+
```
Comment (michaelbahng999) [inline html_helper.js]: Use `const` for module imports

File: html_helper.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+let saveHtmlContent = (fileName, contents) => {
+    fs.writeFileSync(fileName, contents);
+}
+
```
Comment (michaelbahng999) [inline html_helper.js]: use `const` for functions

File: html_helper.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        saveHtmlContent(fileName, "<!DOCTYPE html><html><body></body></html>");
+    }
+    var contents = fs.readFileSync('./test.html', 'utf8').toString();
+    return cheerio.load(contents);
+}
+
+module.exports = {
+    saveHtmlContent: saveHtmlContent,
+    getHtmlContent: getHtmlContent
+}
```
Comment (michaelbahng999) [inline html_helper.js]: ES6: just do module.exports = {saveHtmlContent, getHtmlContent};

File: html_helper.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+let saveHtmlContent = (fileName, contents) => {
+    fs.writeFileSync(fileName, contents);
+}
+
+let getHtmlContent = (fileName) => {
+    if (!fs.existsSync(fileName)) {
+        saveHtmlContent(fileName, "<!DOCTYPE html><html><body></body></html>");
+    }
+    var contents = fs.readFileSync('./test.html', 'utf8').toString();
```
Comment (michaelbahng999) [inline html_helper.js]: favor `const` when you can, you shouldn't use var

**Review by clghua — APPROVED** [review]

File: index.js [hunk 5]
```diff
@@ -1,5 +1,46 @@
+const cheerio = require('cheerio');
+const html_helper = require('./html_helper');
+const NAME_TO_SCHEMA_MAP = require('./name_to_scham_map');
```
Comment (michaelbahng999) [inline index.js]: type

Comment (michaelbahng999) [reply index.js]: *typo

File: index.js [hunk 6]
```diff
@@ -1,5 +1,46 @@
...
+    NAME_TO_SCHEMA_MAP.set(schema.tableName, schema);
+
+    for (let [key, schema] of NAME_TO_SCHEMA_MAP) {
+        if (!localTablesMap.has(schema.tableName)) {
+            let headers = '';
+            console.log(schema.columns);
+            for (let columnName of Array.from(schema.columns)) {
+                headers += `<td>${columnName}</td>`;
+            }
+            $('body').append(`<table id="#${schema.tableName}"><thead><tr>${headers}</tr></thead><tbody></tbody></table>`);
```
Comment (michaelbahng999) [inline index.js:31]: will make separate PR to add whitespace to html output

Comment (michaelbahng999) [reply index.js:31]: https://www.npmjs.com/package/pretty

**Review by michaelbahng999 — APPROVED** [review]

File: html_helper.js (hunk 1 above)
Comment (jastribl) [reply html_helper.js]: done

File: html_helper.js (hunk 2 above)
Comment (jastribl) [reply html_helper.js]: done

File: html_helper.js (hunk 4 above)
Comment (jastribl) [reply html_helper.js]: done

File: html_helper.js (hunk 3 above)
Comment (jastribl) [reply html_helper.js]: done

File: index.js (hunk 5 above)
Comment (jastribl) [reply index.js]: done
